src/components/listItemComponent/ListItemComponent.jsx

```diff
@@ -2,46 +2,139 @@ import React from 'react';
 import styled from 'styled-components';
+import AwardsItemComponent from './AwardsItemComponent';
 
-const TileDiv = styled.div`
-    width: 1000px;
-    height: 100px;
+const TileDivContainer = styled.div`
+	height: 22vh;
     background-color: white;
     margin: 10px;
+	display: flex;
+	max-width: 80vw;
+
+`;
+
+const ThumbnailDiv = styled.div`
+	display: flex;
+	color: black;
+	flex-direction: column;
+	align-items: center;
+	justify-content: center;
+	width: 160px;
+`;
+
+const ContentDiv = styled.div`
+	color: black;
+	display: flex;
+	flex-direction: column;
+	justify-content: space-between;
+	margin: 20px;
+	max-width: 90%;
+`;
+
+const TitleDivContainer = styled.div`
+	display: flex;
+	flex-direction: row;
+	justify-content: flex-start;
+
+
+	max-width: 90%;
+	min-width: 90%
+`;
+
+const AuthorDivContainer = styled.div`
+	color: black;
+	display: flex;
+	flex-direction: row;
+	justify-content: row-start;
+	margin-top: 0px;
+`;
+
+const Permalink = styled.div`
+
+`;
+
+const Footer = styled.div`
+	display: flex;
+	flex-direction: row;
+	align-items: left;
+	justify-content: space-around;
+	max-width: 100%;
+	min-width: 70%;
+`;
+
+const Ups = styled.div`
+	color: green;
+	display: flex;
+	align-items: flex-end;
+`;
+
+const AllAwardingsDiv = styled.div`
+
 `;
 
 const ListItemComponent = () => {
 	const redditApiReturn = useSelector(selectRedditReturn);
-	console.log('redditApiReturn', redditApiReturn);
 
 	if(!redditApiReturn.data) {
-		console.log('api return.data is not true', redditApiReturn.data);
 		return <div>reddit is loading...</div>;
 	} else 
 		return (
 			<div>
-				{console.log('redditApiReturn.data', redditApiReturn.data)}
+
 				{
 					redditApiReturn.data.children.map((each, index) => {
+						let iconUrlArray = [];
+						if(each.data.all_awardings[0]){
+							let allAwardings = each.data.all_awardings;
+
+							allAwardings.map(awarding => {
+								let iconUrl = awarding.resized_icons[2].url;
+								iconUrlArray.push(iconUrl);
+							});
+						}
+
+						const getDate = () => {
+							const date = new Date(each.data.created_utc);
+							return date;
+						};
+
 						const { 
-							// created,
-							// author,
-							// ups,
+							created,
+							author,
+							ups,
 							title,
 							thumbnail,
-							// permalink
+							permalink,
+
 						} = each.data;
-						const defaultThumb = 'gt86nurburgring.jpeg';
+
+						let date = getDate(created).toUTCString();
+
 						return (
 							<div key={index}>
-								<TileDiv>
-									<div style={{display: 'flex', color: 'black'}}>
-										<img src={thumbnail || defaultThumb} alt={title} style={{maxHeight: '100px'}}/>
-										<div>
-											<div >{title}</div>
-										</div>
-									</div>
-								</TileDiv>
+								<TileDivContainer>
+									<ThumbnailDiv>
+										<img src={thumbnail || 'gt86nurburgring.jpg'} alt={title} style={{maxHeight: '80%'}}/>
+									</ThumbnailDiv>
```

Comment:
There are a couple of reasons why this might not be working. To be methodical about it, first make sure your fallback image works. You can do this by not using the thumbnail variable until you figure it out. Just remove thumbnail for now. If you can't figure out how to make it work from the public folder, just move the image file into the same folder as this file, and import it as fallbackImg. That should work for sure.

After you figure that out, you can add the thumbnail variable back and then it'll break again, and you'll be wondering wtf. So console log thumbnail and you'll see the problem. You're probably gonna have to go back to a ternary operator or an if statement maybe higher up to get this to work